meroxa/resource_connector_test.go

```diff
@@ -77,6 +77,67 @@ func TestAccMeroxaConnector_WithoutPipeline(t *testing.T) {
 	})
 }
 
+func TestAccMeroxaConnector_WithInvalidName(t *testing.T) {
```

Comment:
I realize this is only testing invalid cases. Thoughts on me adding another test that is specifically using a "valid" name, or do you think the existing tests with valid names are sufficient?

Reply:
:+1: on adding a valid case